src/service/xhr-impl.js

```diff
@@ -213,7 +213,15 @@ export class Xhr {
    */
   fetchJson(input, opt_init, opt_allowFailure) {
     const init = setupInit(opt_init, 'application/json');
+    const headerContentType = init.headers['Content-Type'];
     if (init.method == 'POST' && !isFormData(init.body)) {
+      const isRequestBodyFormUrlEncoded =
+        (headerContentType ===
```

Comment:
Let's just test that `x-www-form-urlencoded` is in the header. The exact form doesn't matter.

Reply:
>The exact form doesn't matter.

Didn't get this part, i am checking the `content-type` header

Reply:
What I meant is that this could should probably look like this:
```
- if (init.method == 'POST' && !isFormData(init.body)) {
+ const headerContentType = ... || '';
+ if (init.method == 'POST' && !isFormData(init.body) && headerContentType.indexOf('application/x-www-form-urlencoded') == -1) {
```

I.e. we want `headerContentType contains application/x-www-form-urlencoded` case to be treated exactly the same as isFormData(init.body). And we probably just want `indexOf` in case anyone else decides to add something to it.